build.sbt

```diff
@@ -25,16 +25,16 @@ val awsSdkVersion           = "2.17.184"
 val byteBuddyAgentVersion   = "1.10.17"
 val betterMonadicForVersion = "0.3.1"
 val caffeineVersion         = "3.1.8"
-val catsEffectVersion       = "2.5.5"
-val catsRetryVersion        = "2.1.1"
+val catsEffectVersion       = "3.5.2"
+val catsRetryVersion        = "3.1.0"
 val catsVersion             = "2.10.0"
 val circeVersion            = "0.14.6"
 val circeOpticsVersion      = "0.15.0"
 val circeExtrasVersions     = "0.14.3"
 val classgraphVersion       = "4.8.164"
 val distageVersion          = "1.0.10"
```

Comment:
Update distage version to get support for CE3

Reply:
This does help with the class not found problem, thanks